tools/db_crashtest.py

```diff
@@ -348,6 +350,8 @@ def is_direct_io_supported(dbname):
     # pipeline write is not currnetly compatible with WritePrepared txns
     "enable_pipelined_write": 0,
     "create_timestamped_snapshot_one_in": random.choice([0, 20]),
+    # PutEntity in transactions is not yet implemented
+    "use_put_entity_one_in" : 0,
```

Comment:
Do we need to disable it if user-defined timestamp is enabled?

Reply:
I think we already do that: `ts_params` also sets `use_put_entity_one_in` to 0. Please let me know if I missed something.